rosidl_runtime_cpp/docs/FEATURES.md

```diff
@@ -25,3 +25,16 @@ These type traits are useful for providing compile-time checks of rosidl C++ typ
 
 Provided in `bounded_vector.hpp` is an implementation of a std::vector like container class, that has a bounded capacity.
 This class is utilized by rosidl types that declare a bounded sequence.
+
+## Type Description Interface Structs
+
+A copy of the output of `rosidl_generator_cpp` for `type_description_interfaces` messages.
```

Comment:
Same comment; we should add in why the copy is needed at all.

Reply:
Done